src/adapter/src/coord/sequencer/inner.rs

```diff
@@ -3944,7 +3944,7 @@ impl Coordinator {
         };
         for privilege_object in &privilege_objects {
             self.catalog()
-                .ensure_not_reserved_role(&privilege_object.role_id)?;
```

Comment:
For my own knowledge, is the system role the only "reserved" role?

Reply:
Any role with a system ID (`mz_system` and `mz_introspection`) and `PUBLIC` are reserved roles. We also have some prefixes that we consider reserved when creating a new role, however that's not an issue for already existing roles.